arctic/store/_ndarray_store.py

```diff
@@ -409,9 +409,10 @@ def append(self, arctic_lib, version, symbol, item, previous_version, dtype=None
 
         version['type'] = self.TYPE
         version[FW_POINTERS_CONFIG_KEY] = ARCTIC_FORWARD_POINTERS_CFG.name
+
         # Create an empty entry to prevent cases where this field is accessed without being there. (#710)
-        if version[FW_POINTERS_CONFIG_KEY] != FwPointersCfg.DISABLED.name:
-            version[FW_POINTERS_REFS_KEY] = list()
+        if previous_version and previous_version[FW_POINTERS_CONFIG_KEY] != FwPointersCfg.DISABLED.name:
```

Comment:
Is it guaranteed this field is there in previous_version?
If so not clear to me why reset it to empty list... 

No test impact?

Reply:
@jamesblackburn i think there was a test impact. lots of tests failed on travis

Reply:
@jamesblackburn Sorry, I meant to create the WIP pull requests but got distracted by other stuff. Will close this for now.